custom/aaa/static/aaa/spec/models/child.spec.js

```diff
@@ -55,7 +55,7 @@ describe('Child models', function () {
             );
             assert.equal('1', childListModel.id);
             assert.equal('<a href="unified_beneficiary_details/child/1/?month=3&year=2019">test Name</a>', childListModel.name());
-            assert.equal('1 Yr 2 Mon', childListModel.age());
+            assert.equal('2 Yr 8 Mon', childListModel.age());
```

Comment:
the selected date is March 2019, and the dob is April 2017. Why isn't this 1 Yr 11 Mon ?